R/make_quantile_reg.R

```diff
@@ -49,7 +49,10 @@ quantile_reg <- function(mode = "regression", engine = "rq", quantile_levels = 0
 }
 
 make_quantile_reg <- function() {
-  parsnip::set_new_model("quantile_reg")
+  current_env <- get_model_env()
+  if (!any(current_env$models == "quantile_reg")) {
```

Comment:
Perhaps `!("quantile_reg" %in% model_env$models)`. I somehow feel like that's clearer?

Reply:
sure, that seems a reasonable style thing, I literally copy-pasted this from `check_model_exists`, which throws an error instead of giving the boolean for........ reasons??